### PR TITLE
New: Make Emby imdb tags conditional

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
NO

#### Description
Emby does not support imdb- and uses imdb=
https://github.com/TRaSH-/Guides/pull/1386/files
https://github.com/TRaSH-/Guides/issues/1371
https://emby.media/community/index.php?/topic/118056-incorrect-identity-tv-shows/#comment-1245760

#### Todos
- Tests
- Translation Keys (./src/NzbDrone.Core/Localization/Core/en.json)
- [Wiki Updates](https://wiki.servarr.com)

#### Issues Fixed or Closed by this PR

* Fixes N/A

### CODE REVIEW

**Comment by Qstick** [comment]
You probably want to add a test that matches Emby documentation here

`[tmdbid=xxxx]` for tags

File: src/NzbDrone.Core.Test/OrganizerTests/FileNameBuilderTests/IdFixture.cs [hunk 1]
```diff
@@ -66,5 +84,27 @@ public void should_skip_imdb_tag_if_null()
...
+            _namingConfig.MovieFolderFormat = "{Movie Title} {imdb={ImdbId}}";
+
+            _movie.ImdbId = null;
+
+            Subject.GetMovieFolder(_movie)
+                   .Should().Be($"Movie Title");
+        }
+
+        [Test]
+        public void should_skip_tmdbid_tag_if_null_with_equals()
```
Comment (Qstick) [inline src/NzbDrone.Core.Test/OrganizerTests/FileNameBuilderTests/IdFixture.cs:100]: This is impossible, no need to test, switch to IMDb with square brackets

**Review by Qstick — CHANGES_REQUESTED** [review]

**Comment by Qstick** [comment]
Stick to handling IMDb null as was the intention of this PR, TMDb can never be null. The comment above was an example of their format direct from their docs.... Not a recommendation to add tmdbId.

**Comment by bakerboy448** [comment]
cleaned up accordingly.

need guidance on how to have the brackets be conditional.

`{Movie Title} {[imdb=ImdbId]}` did not work
`{Movie Title} [imdb={ImdbId}]` did not work
`{Movie Title} {[imdb={ImdbId}]}` did not work

**Comment by TRaSH-** [comment]
Little bump for this PR, I would like to update the guide to follow the correct weird option Emby uses for their naming, but i can't merge it before this issue is resolved.

**Comment by darthShadow** [comment]
Another option would be to ask Emby to support the Plex-style tags if someone wants to comment here: https://emby.media/community/index.php?/topic/121739-plex-format-id-tags-for-shows-movies/

**Comment by darthShadow** [comment]
Following the above Feature Request, support for Plex-style tags has been added in the latest Emby beta (https://github.com/MediaBrowser/Emby.Releases/releases/tag/4.8.0.48):

```
Support more conventions with provider id values in folder names (moviedb, imdb, etc)
```

If someone can confirm it's working as expected, then this can be closed, I think.

**Pull request closed without merging**